.vscode/settings.json

```diff
@@ -10,9 +10,14 @@
     "statusBar.background": "#1a1f15",
     "statusBarItem.hoverBackground": "#333d2a",
     "statusBar.foreground": "#e7e7e7",
-    "panel.border": "#333d2a",
-    "sideBar.border": "#333d2a",
-    "editorGroup.border": "#333d2a"
+    "commandCenter.border": "#e7e7e799",
```

Comment:
don't change the vscode settings, kindly revert this

Reply:
noted with thanks